README.md

```diff
@@ -5,41 +5,46 @@
 [![GitHub release](https://img.shields.io/github/release/EGI-Foundation/cloud-info-provider.svg)](https://github.com/EGI-Foundation/cloud-info-provider/releases)
 
 The Cloud Information provider generates a representation of cloud resources,
-that can be published inside a BDII (using the provided LDIF templates for a
-GlueSchema v2 representation) or any other component like the INDIGO
-Configuration Management Database (CMDB) (Using specific templates).
+that can be published by different systems, like a BDII (using the provided
+LDIF templates for GlueSchema represntation) or any other component like the
+INDIGO Configuration Management Database (CMDB) (Using specific templates).
 
-The generated representation is described using a
-[Mako](http://www.makotemplates.org/) template having access to the cloud
-middleware information.
+The provider extracts information from a cloud deployment using its public API
+and formats it according to [Mako](http://www.makotemplates.org/) templates.
 
-Supported cloud middleware providers:
+Current supported cloud middleware:
 
-* OpenNebula
-* OpenStack/ooi using Keystone authentication with API v3 *only*
+* OpenNebula/rOCCI
+* OpenStack
+* OpenStack/ooi
 
 ## Installation
 
-### Dependencies
+### From binaries
 
-The cloud-provider depends on PyYAML, which is already included as a dependency for
-binary packages and when installing from source.
+Latest packages are made available at [EGI's AppDB](https://appdb.egi.eu/store/software/cloud.info.provider).
```

Comment:
Maybe we could say that all packages are always attached to the releases in GitHub and after they are made available at the AppDB (which is handy for providing a ready-to-use repo).